airbyte-webapp/src/packages/cloud/components/experiments/SignupSourceDropdown/SignupSourceDropdown.tsx

```diff
@@ -0,0 +1,198 @@
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import React, { useCallback, useMemo, useState } from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+import { components } from "react-select";
+import { MenuListProps } from "react-select";
+
+import { GAIcon } from "components/icons/GAIcon";
+import { ControlLabels } from "components/LabeledControl";
+import {
+  DropDown,
+  DropDownOptionDataItem,
+  DropDownOptionProps,
+  OptionView,
+  SingleValueIcon,
+  SingleValueProps,
+  SingleValueView,
+} from "components/ui/DropDown";
+import { Text } from "components/ui/Text";
+
+import { ReleaseStage } from "core/request/AirbyteClient";
+import { useModalService } from "hooks/services/Modal";
+import RequestConnectorModal from "views/Connector/RequestConnectorModal";
+import styles from "views/Connector/ServiceForm/components/Controls/ConnectorServiceTypeControl/ConnectorServiceTypeControl.module.scss";
+import { useAnalyticsTrackFunctions } from "views/Connector/ServiceForm/components/Controls/ConnectorServiceTypeControl/useAnalyticsTrackFunctions";
+import { WarningMessage } from "views/Connector/ServiceForm/components/WarningMessage";
+
+import { useGetSourceDefinitions } from "./useGetSourceDefinitions";
+import { getSortedDropdownData } from "./utils";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type MenuWithRequestButtonProps = MenuListProps<DropDownOptionDataItem, false> & { selectProps: any };
+
+const ConnectorList: React.FC<React.PropsWithChildren<MenuWithRequestButtonProps>> = ({ children, ...props }) => (
```

Comment:
🧹  I duplicated `<ConnectorServiceTypeControl/>`. If we decide to keep this, all these components can be extracted and reused between both components